lib/profiler/mitmDumpProfiler.py

```diff
@@ -36,8 +38,26 @@ def start_recording(self):
                 replay_cmd_list = ["mitmdump", "--no-http2", "-S", flow_file_fp, "--no-pop", "--norefresh"]
             self.process = subprocess.Popen(replay_cmd_list)
 
+        if sys.platform == "linux2":
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy", "mode", "manual"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.http", "host", "127.0.0.1"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.http", "port", "8080"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.https", "host", "127.0.0.1"])
+            subprocess.call(["gsettings", "set", "org.gnome.system.proxy.https", "port", "8080"])
+        else:
```

Comment:
i'll suggest using `elif 'win32'` and `elif 'darwin'` to check the os, and `else` for handling the other cases.